backend/app/controllers/spree/admin/payment_methods_controller.rb

```diff
@@ -49,23 +49,38 @@ def collection
       end
 
       def load_providers
-        @providers = Spree::PaymentMethod.providers.sort_by(&:name)
+        load_payment_methods
       end
+      deprecate load_providers: :load_payment_methods, deprecator: Spree::Deprecation
 
-      def validate_payment_method_provider
+      def load_payment_methods
+        @payment_methods = Rails.application.config.spree.payment_methods.sort_by(&:name)
```

Comment:
I see. `payment_method_types` is ok, or `configured_payment_methods` could also work, no? @mamhoff thoughts?